### ISSUE TITLE
Support for callback function for `grid.tooltip.formatter`?

### ISSUE THREAD

**Issue by jtpio** [trungleduc/ipecharts#22]
The official Echarts docs mention the use of a callback function for `grid.tooltip.formatter ` as a second option: https://echarts.apache.org/en/option.html#grid.tooltip.formatter

However it's not clear if such functionality could be supported by the `EChartsRawWidget`, since the function is expected to be in JavaScript in the original Echarts library.

Maybe a good tradeoff could be to allow passing a JS function as a raw string, to be evaluated and executed on the frontend.

**Comment by trungleduc** [comment]
Yep, supporting JS callback is quite tricky, even by using the raw JS string since most JS classes are auto-generated. 
If the number of callback options is not too much, I could do it manually, but it is not on my priority list. For the next version, I want to focus on supporting chart action (https://echarts.apache.org/en/api.html#action)

**Comment by trungleduc** [comment]
https://github.com/trungleduc/ipecharts?tab=readme-ov-file#using-js-function-in-chart-options

**Comment by jtpio** [comment]
Nice, thanks!
